### ISSUE TITLE
AttributeError with python 3.6.9 on Ubuntu 18.04

### ISSUE THREAD

**Issue by Void9711** [azuline/pixiv-api#6]
When I tried to run example code in https://pixiv-api.readthedocs.io/en/latest/, I got an `AttributeError` as follow:

> Traceback (most recent call last):
  File "pixiv_works_download.py", line 9, in <module>
    illustration = client.fetch_illustration(75523989)
  File "/home/void/.local/lib/python3.6/site-packages/pixivapi/common.py", line 25, in wrapper
    return func(self, *args, **kwargs)
  File "/home/void/.local/lib/python3.6/site-packages/pixivapi/client.py", line 251, in fetch_illustration
    client=self,
  File "/home/void/.local/lib/python3.6/site-packages/pixivapi/models.py", line 240, in __init__
    self.create_date = datetime.fromisoformat(create_date)
AttributeError: type object 'datetime.datetime' has no attribute 'fromisoformat'

My code:
```python
from pixivapi import Client
from pathlib import Path
from pixivapi import Size

client = Client()

client.login('<my_username>', '<my_password>')

illustration = client.fetch_illustration(75523989)
illustration.download(
    directory=Path.home() / 'my_pixiv_images',
    size=Size.ORIGINAL,
)
```

**Comment by sentouki** [comment]
`date.fromisoformat` is available since python3.7 , source: [python docs](https://docs.python.org/3/library/datetime.html#datetime.date.fromisoformat)
the easiest solution would be an upgrade of python

**Comment by azuline** [comment]
resolved in pr #7
